### ISSUE TITLE
IBAN position on the payment part

### ISSUE THREAD

**Issue by popod** [schoero/SwissQRBill#350]
Hi,

Since v3, I think that the IBAN position (on the payment part) is really to far of it's title.. Why have you done this change ?

--
![test](https://user-images.githubusercontent.com/3807312/141298943-60b101bc-80a3-4bd2-9815-4594baee6481.jpg)

--

If possible, I think that this will be really better to add a more bigger margin (1-2 mm) between "Compte / Payable à" and the IBAN.

Thanks.

**Comment by schoero** [comment]
> Why have you done this change ?

Well, the intention was to match the [specifications](https://www.paymentstandards.ch/dam/downloads/style-guide-en.pdf) (Described on page 15).
The spacing in v2 was definitely too large, but you are right that the spacing in the current version is too small. 

<img width="985" alt="Screenshot 2021-11-11 at 19 34 24" src="https://user-images.githubusercontent.com/42278642/141351301-e2a9cf2a-9872-42b9-9c84-5ea61e688a1d.png">

Actually, I have noticed that the spacing of most of the text fields is quite off when I have overlapped the image from the specifications with a generated bill.

I try to fix these issues by getting as close to the specifications as possible.
